pkg/api/server/test/db.go

```diff
@@ -60,6 +60,9 @@ func NewDB(t *testing.T) *gorm.DB {
 		t.Fatalf("failed to open the results.db: %v", err)
 	}
 
+	// Enable foreign key support
+	gdb.Exec("PRAGMA foreign_keys = ON;")
```

Comment:
does this need to go into the SQL file too?

Reply:
No. This is something that's only needed for sqlite, so it only needs to be applied for tests (where as the .sql file is used by both sqlite and mysql).

Reply:
FWIW I'm not a big fan of this split usage of databases for tests vs deploy, but AFAICT there's no easy way to spin up a local mysql instance so it's the best we have at the moment for local testing with a real DB.

Reply:
yeah that's unfortunate. I saw https://github.com/proullon/ramsql though that seems unmaintained.